From d2513b66ff68f72517dfa893794524ec623b588b Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Mon, 24 Jul 2023 01:29:55 +0200
Subject: [PATCH 458/464] leds: Add orange color

It's already part of DT bindings. Make the kernel be able to convert
it to string.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 drivers/leds/led-core.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/leds/led-core.c b/drivers/leds/led-core.c
index b9b1295833c9..042e2995eb37 100644
--- a/drivers/leds/led-core.c
+++ b/drivers/leds/led-core.c
@@ -33,6 +33,7 @@ const char * const led_colors[LED_COLOR_ID_MAX] = {
 	[LED_COLOR_ID_AMBER] = "amber",
 	[LED_COLOR_ID_VIOLET] = "violet",
 	[LED_COLOR_ID_YELLOW] = "yellow",
+	[LED_COLOR_ID_ORANGE] = "orange",
 	[LED_COLOR_ID_IR] = "ir",
 	[LED_COLOR_ID_MULTI] = "multicolor",
 	[LED_COLOR_ID_RGB] = "rgb",
-- 
2.34.1

